README.md

```diff
@@ -115,38 +108,38 @@ With J2V all the structures are discovered automatically and two files are gener
 
 SELECT
 
----chains_table Information
-chains_table."raw_data_column":"apiVersion"::string AS APIVERSION
-,chains_table."raw_data_column":"data Provider"::string AS DATA_PROVIDER
-,chains_table."raw_data_column":"dataGenerationTimestamp"::string AS DATAGENERATIONTIMESTAMP
-,chains_table."raw_data_column":"headquater":"building":"address"::string AS HEADQUATER_BUILDING_ADDRESS
-,chains_table."raw_data_column":"headquater":"city"::string AS HEADQUATER_CITY
-,chains_table."raw_data_column":"headquater":"country"::string AS HEADQUATER_COUNTRY
-,chains_table."raw_data_column":"headquater":"employees"::number AS HEADQUATER_EMPLOYEES
-,chains_table."raw_data_column":"payloadPrimaryKeyValue"::string AS PAYLOADPRIMARYKEYVALUE
+---JSON_TABLE Information
+IFNULL(JSON_TABLE."DATA":"apiVersion"::string,'N/A') AS APIVERSION
+,IFNULL(JSON_TABLE."DATA":"data Provider"::string,'N/A') AS DATA_PROVIDER
+,IFNULL(JSON_TABLE."DATA":"headquarter":"building":"address"::string,'N/A') AS HEADQUARTER_BUILDING_ADDRESS
+,IFNULL(JSON_TABLE."DATA":"headquarter":"city"::string,'N/A') AS HEADQUARTER_CITY
+,IFNULL(JSON_TABLE."DATA":"headquarter":"country"::string,'N/A') AS HEADQUARTER_COUNTRY
+,IFNULL(JSON_TABLE."DATA":"headquarter":"employees"::number,0) AS HEADQUARTER_EMPLOYEES
+,IFNULL(JSON_TABLE."DATA":"payloadPrimaryKeyValue"::string,'N/A') AS PAYLOADPRIMARYKEYVALUE
+,IFNULL(JSON_TABLE."DATA":"version"::string,'N/A') AS VERSION
+,JSON_TABLE."DATA":"dataGenerationTimestamp"::timestamp AS DATAGENERATIONTIMESTAMP
 ,
 ---restaurants Information
-restaurants.VALUE:"address"::string AS RESTAURANTS_ADDRESS
-,restaurants.VALUE:"city"::string AS RESTAURANTS_CITY
-,restaurants.VALUE:"country"::string AS RESTAURANTS_COUNTRY
-,restaurants.VALUE:"currency"::string AS RESTAURANTS_CURRENCY
-,restaurants.VALUE:"name"::string AS RESTAURANTS_NAME
+IFNULL(restaurants.VALUE:"address"::string,'N/A') AS RESTAURANTS_ADDRESS
```

Comment:
Was this example created with the IFNULL flag? Should it be created without it to be consistent with the previous example? Or if we want to change the example's output, can we do it in a separate pull request?